deploy/example_catalog/cr-app-datafabric610.json

```diff
@@ -0,0 +1,232 @@
+{
+  "apiVersion": "kubedirector.hpe.com/v1beta1",
+  "kind": "KubeDirectorApp",
+  "metadata": {
+    "name": "datafabric610"
+  },
+  "spec": {
+    "systemdRequired": true,
+    "defaultPersistDirs": [
+      "/opt/mapr"
+    ],
+    "config": {
+      "configMeta": {
+        "storage": "60",
+        "secure": "true"
+      },
+      "roleServices": [
+        {
+          "roleID": "control-system",
+          "serviceIDs": [
+            "warden",
+	    "zookeeper",
+            "yarn-hs",
+            "mapr-cs"
+          ]
+        },
+        {
+          "roleID": "cldb",
+          "serviceIDs": [
+            "warden",
+	    "zookeeper",
+            "cldb"
+          ]
+        },
+        {
+        "roleID": "oozie",
+        "serviceIDs": ["warden", "oozie", "mysql"]
+      },
+        {
+          "roleID": "edge",
+          "serviceIDs": [
+            "mapr-client",
```

Comment:
mapr-client is a service?

Reply:
mapr-client is actually a package and not a service. I have mentioned it so that it wont be blank.